submit/controllers/process.py

```diff
@@ -304,10 +305,20 @@ def file_preview(params, session: Session, submission_id: int, token: str,
                  **kwargs) -> Response:
     submitter, client = user_and_client_from_session(session)
     submission, submission_events = load_submission(submission_id)
-    prod = Compiler.get_product(submission.source_content.identifier,
-                                submission.source_content.checksum, token)
-    headers = {'Content-Type': prod.content_type}
-    return prod.stream, status.OK, headers
+    if (submission.source_content.source_format == SubmissionContent.Format.PDF):
+        subfiles = FileManager.get_upload_status(
+            submission.source_content.identifier,
+            token)
+        pdf_name = next(( file.name for file in subfiles.files if file.file_type=='PDF'))
+        (pdf_dload,rh) = FileManager.get_file_content(submission.source_content.identifier,
+                                            pdf_name, token)
```

Comment:
Minor: watch out for spacing around commas, line length, and indentation of continuation lines. Since this is kind of a long of a long one, consider something like:

```python
pdf_dload, rh = \
    FileManager.get_file_content(submission.source_content.identifier, 
                                 pdf_name, token)
```